.github/workflows/release.yml

```
@@ -6,7 +6,7 @@ permissions:
 
 on:
   push:
-    branches: ["main"]
+    branches: [ main ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)